app/models/issues_data_series.rb

```diff
@@ -0,0 +1,22 @@
+class IssuesDataSeries
+
+  attr_reader :open_issues, :closed_issues
+
+  def initialize open_issues, closed_issues
+    @open_issues = open_issues
+    @closed_issues = closed_issues
+  end
+
+  def series
+    [
+      {
+        name: 'Open Issues',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.